fairer-iOS/fairer-iOS/Screens/Home/SettingHomeRule/ViewController/SettingHomeRuleViewController.swift

```diff
@@ -9,10 +9,14 @@ import UIKit
 
 import SnapKit
 
+import Moya
+
 class SettingHomeRuleViewController: BaseViewController {
```

Comment:
p3;
```suggestion
final class SettingHomeRuleViewController: BaseViewController {
```
상속 받을 일 없어서 final 추가 부탁드려요!

Reply:
https://github.com/fairer-iOS/fairer-iOS/pull/112/commits/f638deb5ffe8a664ed3e4a62c42ce1e3659e2a33